static/app/views/performance/newTraceDetails/traceDrawer/details/durationComparison.tsx

```diff
@@ -0,0 +1,68 @@
+import styled from '@emotion/styled';
+
+import {Row} from 'sentry/components/events/interfaces/spans/newTraceDetailsSpanDetails';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {getDuration} from 'sentry/utils/formatters';
+
+type Props = {
+  duration: number;
+  title: string;
+  avgDuration?: number;
```

Comment:
```suggestion
  baseline?: number;
```

that way you can use the same component for rendering pxx values in the future